test/parallel/test-fs-timestamp-parsing-error.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const fs = require('fs');
+const assert = require('assert');
+require('../common');
+
+[Number.POSITIVE_INFINITY, Number.NEGATIVE_INFINITY, NaN].forEach((input) => {
+  assert.throws(() => fs._toUnixTimestamp(input),
+                new RegExp('^Error: Cannot parse time: ' + input + '$'));
+});
+
+const okInputs = [1, -1, '1', '-1', Date.now()];
+okInputs.forEach((input) => assert.doesNotThrow(() =>
```

Comment:
I think using braces and better indentation for the `forEach()` callback will make this look better (especially since `assert` does not return meaningful values). Example:

```js
okInputs.forEach((input) => {
  assert.doesNotThrow(() => fs._toUnixTimestamp(input));
});
```